src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+import { User } from './services/User';
+import { Todo } from './services/Todo';
+import { TodoWithUser } from './services/TodoWithUser';
+
+function getUserById(id: number): User | undefined {
+  return usersFromServer.find(user => user.id === id);
+}
+
+function newListWithUser(): TodoWithUser[] {
```

Comment:
```suggestion
function getTodosWithUser(): TodoWithUser[] {
```
A function should contain a verb, otherwise it seems like `newListWithUser` variable is already prepared list. Also, it's better to be more precise - it's array of todos, "list" word says nothing here